### PR TITLE
An "Update-sized update"

### PR DESCRIPTION
**Pull request by archuserbtw926**
- adding a bug so i can fix it later and make it look like i am productive!!!
- lowered wait times

### CODE REVIEW

**Comment by Sneak1000** [comment]
Piss cum and shit

**Comment by archuserbtw926** [comment]
nvm meant to write "holiday-sized update"

File: among-us.py [hunk 1]
```diff
@@ -1,6 +1,14 @@
 # need devs
 
+import random
 import time
+
 print("please wait")
```
Comment (Sneak1000) [inline among-us.py:6]: How about you print some more bitches

Comment (archuserbtw926) [reply among-us.py:6]: wtf reported my feelings were hurt

File: among-us.py [hunk 2]
```diff
@@ -1,6 +1,14 @@
 # need devs
 
+import random
```
Comment (Sneak1000) [inline among-us.py:3]: How about you import some more bitches

File: among-us.py [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
 import time
+
 print("please wait")
-time.sleep(3599)
-print("please try again")
+time.sleep(3587)
+
+lol=random.randrange(1, 10)
+if lol==2:
```
Comment (Sneak1000) [inline among-us.py:10]: It would be nice IF you could get some more bitches

File: among-us.py (hunk 2 above)
Comment (archuserbtw926) [reply among-us.py:3]: bro stop im gonna cry

File: among-us.py [hunk 4]
```diff
@@ -1,6 +1,14 @@
...
+    print("Error: Try running this program again.")
+    hehehehaw=0/0
+else:
```
Comment (Sneak1000) [inline among-us.py:13]: else:
    You get no bitches

File: among-us.py (hunk 3 above)
Comment (archuserbtw926) [reply among-us.py:10]: NAH ITS OVER. FIGHT ME IM WAITING. COME TO THE PIZZA HUT WE HAD A DATE AT LAST WEEK.

File: among-us.py (hunk 4 above)
Comment (archuserbtw926) [reply among-us.py:13]: prove it kid

File: among-us.py [hunk 5]
```diff
@@ -1,6 +1,14 @@
 # need devs
 
+import random
 import time
+
 print("please wait")
-time.sleep(3599)
```
Comment (Sneak1000) [inline among-us.py:5]: It's TIME for you to get more bitches

File: among-us.py [hunk 6]
```diff
@@ -1,6 +1,14 @@
...
-print("please try again")
+time.sleep(3587)
+
+lol=random.randrange(1, 10)
+if lol==2:
+    print("Error: Try running this program again.")
+    hehehehaw=0/0
+else:
+    print("Error: Operation Successful")
```
Comment (Sneak1000) [inline among-us.py:14]: How about you start an operation with some more bitches

Comment (archuserbtw926) [reply among-us.py:14]: bro theres only 14 lines of code im sorry pls stop

File: among-us.py [hunk 7]
```diff
@@ -1,6 +1,14 @@
...
 import time
+
 print("please wait")
-time.sleep(3599)
-print("please try again")
+time.sleep(3587)
+
+lol=random.randrange(1, 10)
+if lol==2:
+    print("Error: Try running this program again.")
```
Comment (Sneak1000) [inline among-us.py:11]: Error: No bitches detected

File: among-us.py [hunk 8]
```diff
@@ -1,6 +1,14 @@
 # need devs
```
Comment (Sneak1000) [inline among-us.py:1]: You NEED to get more bitches

File: among-us.py (hunk 7 above)
Comment (archuserbtw926) [reply among-us.py:11]: Error: you still havent showed up to the pizza hut. KID DONT KEEP ME WAITING IM NOT WASTING GAS COMING TO YOU KID

File: among-us.py (hunk 1 above)
Comment (Sneak1000) [inline among-us.py:6]: Please don't make me wait for you to get more bitches

File: among-us.py (hunk 8 above)
Comment (archuserbtw926) [reply among-us.py:1]: new pie chart
u make up 11% of all greenhouse gases kid

File: among-us.py (hunk 1 above)
Comment (archuserbtw926) [reply among-us.py:6]: HOW ABOUT YOU MAKE ME STOP WAITING FOR YOU TO SHOW UP TO THE PIZZA HUT KID

**Comment by archuserbtw926** [comment]
> Piss cum and shit

unfathomably based

File: among-us.py (hunk 8 above)
Comment (Sneak1000) [reply among-us.py:1]: M1A2 Abrams MBT
I am outside your house kid with an American M1A2 Abrams Main Battle Tank Manufactured in the USA loaded with Amor-piercing High-Explosive Fin-Stabilized Discarding Sabot High-Explosive Anti-Tank High-Explosive Capped Ballistic Capped shells

Comment (archuserbtw926) [reply among-us.py:1]: trollwalk
check ur mailbox

**Pull request merged**